server/app/services/languageProcessor/Processor.java

```diff
@@ -33,65 +33,87 @@ public static String processQuestion(String question) throws IOException,
 			InvocationTargetException,IllegalAccessException {
 		
 		DecisionTree dt = new DecisionTree();
-
-			// Create the Stanford CoreNLP pipeline
-			Properties props = PropertiesUtils.asProperties("annotators", "tokenize,ssplit,pos");
-			StanfordCoreNLP pipeline = new StanfordCoreNLP(props);
-
-			// Annotate an example statement.
-			System.out.println("Question:");
-			System.out.println(question);
-			Annotation ann = new Annotation(question);
-			pipeline.annotate(ann);
-
-			//POS tagging
-			System.out.println("\nPOS tagging:");
-			HashMap<String, String> posTagging = posTagging_(ann);
-
-			//Noun extraction (For now we assume that only one noun is present)
-			String keyword = infoExtraction(posTagging, "NN").get(0);
-
-			/*
-			 * check for Wh-pronoun (WP) - Who, what,
-			 * then for Wh-adverb (WRB)  - when/ where,
-			 * and then Wh-determiner (WDT)  - which
-			 */
-			String ques = null;
-			if (infoExtraction(posTagging, "WP").size() != 0)
-				ques = infoExtraction(posTagging, "WP").get(0).toLowerCase();
-			else if (infoExtraction(posTagging, "WRB").size() != 0)
-				ques = infoExtraction(posTagging, "WRB").get(0).toLowerCase();
-			else if (infoExtraction(posTagging, "WDT").size() != 0)
-				ques = infoExtraction(posTagging, "WDT").get(0).toLowerCase();
-
-			//Create an arrayList for keywords
-			ArrayList<String> keywords_found = new ArrayList<String>();
-
-		  //Analysis of the question
-			System.out.println("\nAnalysis of the question:");
-			String topic = QuestionTopicMapping(keyword).toLowerCase();
-			String keyy = QuestionTypeMapping(ques).toLowerCase();
-
-			keywords_found.add(ques);
-			keywords_found.add(topic);
-			keywords_found.add(keyy);
-			keywords_found.add(keyword);
-
-			System.out.println("\nKeywords found:");
-			System.out.println(keywords_found.toString());
-
-			keywords_found = QuestionMapping(keywords_found);
-
-			System.out.println("\nFinal keyword list:");
-			System.out.println(keywords_found.toString());
-
-			System.out.println("\nQuestion mapping: ");
-			System.out.println(dt.traverse(keywords_found) + "(" + keyword + ")");
-
-
-      String answer = TaskMap.questionMapping(dt.traverse(keywords_found), keyword);
-
-      return answer;
+    final String error_msg = "Sorry, I cannot understand the question!";
+
+    // Create the Stanford CoreNLP pipeline
+    Properties props = PropertiesUtils.asProperties("annotators", "tokenize,ssplit,pos");
+    StanfordCoreNLP pipeline = new StanfordCoreNLP(props);
+
+    // Annotate an example statement.
+    System.out.println("Question:");
+    System.out.println(question);
```

Comment:
Make both of loggers in one line, `System.out.println("Question: " + question);`